pkg/registry/apis/scope/find_scoped_dashboards.go

```diff
@@ -0,0 +1,96 @@
+package scope
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	scope "github.com/grafana/grafana/pkg/apis/scope/v0alpha1"
+)
+
+type findScopedDashboardsREST struct {
+	scopeDashboardStorage *storage
+}
+
+var (
+	_ rest.Storage              = (*findScopedDashboardsREST)(nil)
+	_ rest.SingularNameProvider = (*findScopedDashboardsREST)(nil)
+	_ rest.Connecter            = (*findScopedDashboardsREST)(nil)
+	_ rest.Scoper               = (*findScopedDashboardsREST)(nil)
+	_ rest.StorageMetadata      = (*findScopedDashboardsREST)(nil)
+)
+
+func (f *findScopedDashboardsREST) New() runtime.Object {
+	return &scope.FindScopedDashboardsResults{}
+}
+
+func (f *findScopedDashboardsREST) Destroy() {}
+
+func (f *findScopedDashboardsREST) NamespaceScoped() bool {
+	return true
+}
+
+func (f *findScopedDashboardsREST) GetSingularName() string {
+	return "FindScopedDashboardsResult" // not sure if this is actually used, but it is required to exist
+}
+
+func (f *findScopedDashboardsREST) ProducesMIMETypes(verb string) []string {
+	return []string{"application/json"} // and parquet!
+}
+
+func (f *findScopedDashboardsREST) ProducesObject(verb string) interface{} {
+	return &scope.FindScopedDashboardsResults{}
+}
+
+func (f *findScopedDashboardsREST) ConnectMethods() []string {
+	return []string{"GET"}
+}
+
+func (f *findScopedDashboardsREST) NewConnectOptions() (runtime.Object, bool, string) {
+	return nil, false, "" // true means you can use the trailing path as a variable
+}
+
+func (f *findScopedDashboardsREST) Connect(ctx context.Context, name string, opts runtime.Object, responder rest.Responder) (http.Handler, error) {
+	// See: /pkg/apiserver/builder/helper.go#L34
+	// The name is set with a rewriter hack
+	if name != "name" {
```

Comment:
This doesn't work currently as it has to match `/find$`

Reply:
```
FindScopedDashboardsResults

GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/find_scope_dashboard/{name}
```

and 

```
TreeResults


GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/find
```

are inconsistent in that the URL for ScopedDashboards has name of what it is returning in the path, but `find` is for scopenodes, but doesn't have that in the path.

@ryantxu / @toddtreece Any thoughts on what a good pattern might be here to make these consistent?

Reply:
Maybe something like:

```
ScopedDashboardsResults
GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/scoped_dashboard_results/find/{name}
and

TreeResults
GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/tree_results/find/{name}
```

- This drops "Find" on the type name for one of them, uses underscore word separators, and has the separated return type for find before "find" in the url, and matches the current rewrite regex.

Reply:
another couple options:
* `/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/views/node_tree`
* `/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/views/dashboards`

Reply:
I like the "views" prefix as that makes sense to me, but I'm also okay with find. This adds a view that is returning ScopeDashboardBinding , so I don't think we would want to call it dashboards., maybe dashboard_bindings.